sdk/iothub/azure-resourcemanager-iothub/README.md

```diff
@@ -74,13 +74,15 @@ See [API design][design] for general introduction on design and key concepts on
 
 ## Examples
 
+
+
 ## Troubleshooting
 
 ## Next steps
 
 ## Contributing
 
-For details on contributing to this repository, see the [contributing guide](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).
+For details on contributing to this repository, see the [contributing guide](https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md).
```

Comment:
Please revert this line.

It is a bug in codegen, I will release 4.0.32 to fix it.

Reply:
```suggestion
For details on contributing to this repository, see the [contributing guide](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).
```

Reply:
4.0.32 released